setup.py

```diff
@@ -20,10 +21,17 @@ def get_long_desc():
     desc += open('CHANGES').read()
     return desc
 
+# foolproof way of single-sourcing version as per
+# http://stackoverflow.com/a/24517154/112943
+version_namespace = {}
+version_path = convert_path('bleach/version.py')
+with open(version_path) as version_file:
+    exec(version_file.read(), version_namespace)
```

Comment:
Fixed! Also, there seem to be some build failures due to `flake8` that affect recent PRs. Shall I include the `# flake8: noqa` comment to make it pass here too?